aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -1022,30 +1022,29 @@ static dispatch_data_t getSectionData(const std::string& name) {
   const uint32_t nDim = iter.ndim();
   constexpr uint32_t nOffsets = 3;
   const uint32_t numThreads = iter.numel();
+  const auto suffix = iter.is_contiguous() ? "dense" : "strided";
```

Comment:
does `iter.is_contiguous()` imply that there is no broadcasting involved?

Reply:
That's something I'm unclear about, because `tensors.is_contigous()` returns false for transposed tensors, but `iter.is_contigous()` returns true